### PR TITLE
Bug 1930220: Add udev to the driver image

### PR DESCRIPTION
**Pull request by jsafrane**
NodeStage / NodePublish need to call "udevadm trigger".

### CODE REVIEW

**Comment by jsafrane** [comment]
/hold
pending some testing

**Comment by jsafrane** [comment]
/hold cancel
It seems to have helped without any extra HostPath volumes in the CSI driver pods.

File: images/cinder-csi-plugin/Dockerfile.openshift.ci [hunk 1]
```diff
@@ -1,13 +1,13 @@
-FROM registry.svc.ci.openshift.org/openshift/builder:rhel-8-golang-openshift-4.6
+FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.15-openshift-4.8
```
Comment (mandre) [inline images/cinder-csi-plugin/Dockerfile.openshift.ci:1]: Hmm, shouldn't the ART bot have submitted these changes?

**Review by mandre — COMMENTED** [review]
/lgtm

File: images/cinder-csi-plugin/Dockerfile.openshift.ci (hunk 1 above)
Comment (jsafrane) [reply images/cinder-csi-plugin/Dockerfile.openshift.ci:1]: oh, these are my debug changes to actually build the image. IMO, neither ART nor CI uses `Dockerfile.openshift.ci` and it *could* be removed

**Pull request merged**